zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -154,13 +154,16 @@ impl Config {
         });
 
         meta.create_gate(
-            "limb_difference is equal to the difference at claimed limb",
+            "limb differences before first_different_limb are all 0",
             |meta| {
                 let selector = meta.query_fixed(selector, Rotation::cur());
                 let cur = Queries::new(meta, keys, Rotation::cur());
                 let prev = Queries::new(meta, keys, Rotation::prev());
                 let limb_difference = meta.query_advice(limb_difference, Rotation::cur());
 
+                // E.g. if first_different_limb = 5, four limb differences before need to be 0.
+                // Using RLC, we check that (cur_1 - prev_1) + r(cur_2 - prev_2) + r^2(cur_3 -
+                // prev_3) + r^3(cur_4 - prev_4) = 0.
```

Comment:
Should be at the previous gate?

Reply:
I've unresolved this conversation because I still see the comment, which should go to the previous gate.

Reply:
i forgot to push :|